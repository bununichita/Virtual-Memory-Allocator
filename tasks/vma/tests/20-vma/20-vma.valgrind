==5451== Memcheck, a memory error detector
==5451== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==5451== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==5451== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==5451== Parent PID: 5450
==5451== 
==5451== 
==5451== HEAP SUMMARY:
==5451==     in use at exit: 0 bytes in 0 blocks
==5451==   total heap usage: 10 allocs, 10 frees, 8,377 bytes allocated
==5451== 
==5451== All heap blocks were freed -- no leaks are possible
==5451== 
==5451== For lists of detected and suppressed errors, rerun with: -s
==5451== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
