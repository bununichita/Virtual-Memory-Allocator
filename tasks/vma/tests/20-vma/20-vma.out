Invalid address for free
1
4
